src/test/java/temporary/tests/MainPageProductTest.java

```diff
@@ -0,0 +1,24 @@
+package temporary.tests;
+
+
+import org.junit.Test;
+import pages.mainPage.MainPageProduct;
+import pages.mainPage.MainPageSection;
+import enums.MainPageSectionId;
+import static steps.BaseSteps.openPage;
+
+public class MainPageProductTest {
```

Comment:
Test class should extend `BaseTest` class, to be able to configure these methods (POM instance initialization / quitting web driver instance after the test is done):
![image](https://github.com/fnfldx/automationteststore/assets/125667144/75438649-57f0-4214-bdb5-321b0baef1fe)